test/e2e-tests.sh

```diff
@@ -50,6 +50,16 @@ parallelism=""
 use_https=""
 if (( MESH )); then
   parallelism="-parallel 1"
+
+  echo "Configuring mesh pod addressability"
+  kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-network --type=merge \
```

Comment:
just to check: is there any way to do this pre-install so we don't have to restart the activator/autoscaler?

also will modifying the existing mesh leg rather than adding a new leg mean we lose coverage for changes like #11174 (and the corresponding activator change Im about to land actually)?

Reply:
hmm, now that I think about it, I guess we can mangle the YAML before it gets applied. Ugly, but it would work.

Good point about the leg coverage. I guess we'd need a new leg for this one :thinking: